File: docs/data-tools/codesnippet/CSharp/save-data-in-a-transaction_2.cs
        private void DeleteOrders()
        {
            NorthwindDataSet.OrdersDataTable deletedOrders;
            deletedOrders = (NorthwindDataSet.OrdersDataTable)
                northwindDataSet.Orders.GetChanges(DataRowState.Deleted);

            if (deletedOrders != null)
            {
                try
                {
                    ordersTableAdapter.Update(deletedOrders);
                }
                catch (System.Exception ex)
                {
                    MessageBox.Show("DeleteOrders Failed");
                }
            }
        }

File: docs/data-tools/codesnippet/VisualBasic/save-data-in-a-transaction_2.vb
    Private Sub DeleteOrders()

        Dim deletedOrders As NorthwindDataSet.OrdersDataTable
        deletedOrders = CType(NorthwindDataSet.Orders.GetChanges(Data.DataRowState.Deleted),
            NorthwindDataSet.OrdersDataTable)

        If Not IsNothing(deletedOrders) Then
            Try
                OrdersTableAdapter.Update(deletedOrders)

            Catch ex As Exception
                MessageBox.Show("DeleteOrders Failed")
            End Try
        End If
    End Sub